test/unit/Reflection/ReflectionFunctionAbstractTest.php

```diff
@@ -71,25 +68,6 @@ public function testExportThrowsException() : void
         ReflectionFunctionAbstract::export();
     }
 
-    public function testPopulateFunctionAbstractThrowsExceptionWithInvalidNode() : void
```

Comment:
No longer possible, as the type is enforced by the type hint. The reflectionish approach of the test is no longer valid either.